_data/simple-icons.json

```
@@ -3639,6 +3644,6 @@
             "title": "Zulip",
             "hex": "52C2AF",
             "source": "https://github.com/zulip/zulip/"
-        }
+        }        
```

Just to not litter the commit history, can you remove the whitespace at the end of this line?

```suggestion
        }
```